src/Components/firestore-access.js

```diff
@@ -0,0 +1,293 @@
+'use strict';
+
+//import Investment from './investment';
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+
+const STARTING_MONEY = 10000;
+
+export const setUpRoom =  async (db,NumOfSymbols,Rounds,userID) => {
+  const roomRef = await db.collection('Rooms').doc();
+  const roomID = roomRef.id;
+
+  // TODO: send POST request to get time series data for GOOG
+  // for prototype, hard-code 3 points in time
+  const symbolsL = await initSymbols(db,null,null,NumOfSymbols)
+  console.log("Symbol List is  " + symbolsL)
+  const datesD = await initDates(db,symbolsL,Rounds)
+  roomRef.set({
+    symbols: symbolsL,
+    day_index: 0,
+    dates: datesD["dates"]
+  });
+  
+  await initializeQuiz(symbolsL,roomID,datesD["period"],datesD["dates"])
+
+  const usersRef = roomRef.collection('users');
+  const userRef = usersRef.doc(userID);
+  console.log("SetUp room was called")
+
+
+  const gameInfo = {
+    investments: [],
+    personal_value: STARTING_MONEY,
+    money_left: STARTING_MONEY,
+    gains: 0,
+    losses: 0,
+  }
+  fetch(userRef.set(gameInfo));
+  return roomID;
+}
+
+export const getUserShares = async (db, roomID, userID) => {
+  const userRef = db.collection('Rooms').doc(roomID)
+    .collection('users').doc(userID);
+
+  const userDoc = await userRef.get();
+  const userData = userDoc.data();
+  return userData.investments;
+}
+
+export const getUserBalance = async (db, roomID, userID) => {
+  const userRef = db.collection('Rooms').doc(roomID)
+    .collection('users').doc(userID);
+
+  const userDoc = await userRef.get();
+  const userData = userDoc.data();
+  return userData.money_left;
+}
+
+export const getChartUrl = async(db,roomId,symbol,endDate) =>{
```

Comment:
```suggestion
export const getChartUrl = async (db,roomId,symbol,endDate) =>{
```
nit: add space after async